devsetup/tripleo/net_config.j2

```diff
@@ -14,7 +14,10 @@ network_config:
   - ip_netmask: {{ ctlplane_ip }}/{{ ctlplane_cidr }}
   - ip_netmask: {{ ctlplane_ip }}/32
   - ip_netmask: {{ ctlplane_vip }}/32
-  routes: []
+  routes:
```

Comment:
@bogdando this bit is what is causing the undercloud node to become unreachable

Reply:
this causes a restart of the `eth0` interface, that has the public ip that zuul assigns to the node